src/pages/Challenge/Welcome.js

```diff
@@ -77,12 +79,16 @@ function Welcome({ setActivePage, authenticate, isAuthenticated }) {
       </div>
       <div className="content-section">
         {!isAuthenticated && (
-          <div id="login" className="btn" onClick={() => authenticate()}>
+          <div
+            id="login"
+            className="btn"
+            onClick={() => history.push('/challenge/info')}
```

Comment:
This should call the `authenticate()` function to connect your wallet. 

The button below that says 'Face Your Challenge' should be doing the `history.push('/challenge/info')`